api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeSpec.java

```diff
@@ -50,8 +50,8 @@ public class KafkaBridgeSpec extends Spec implements HasConfigurableLogging, Has
     private String bootstrapServers;
     private ClientTls tls;
     private KafkaClientAuthentication authentication;
-    private KafkaBridgeConsumerSpec consumer;
-    private KafkaBridgeProducerSpec producer;
+    private KafkaBridgeConsumerSpec consumer = new KafkaBridgeConsumerSpec();
+    private KafkaBridgeProducerSpec producer = new KafkaBridgeProducerSpec();
```

Comment:
why the need for this creation here?

Reply:
@ppatierno We have two options, either initialising by default, so `enabled` will be true for both of them, or initialise to `null` and then default it to `true` in the `KafkaBridgeCluster` code `getEnvVars()`.